### PR TITLE
Improved the content in HTML

### PR DESCRIPTION
**Pull request by stealzit**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -95,7 +96,7 @@ <h3 class="features_h3">Elite Clientele</h3>
             </div>
             <div class="col-lg-4 feature_box">
                 <i class="fas fa-heart icon2 fa-4x"></i>
-                <h3 class="features_h3">Guaranteed to work.</h3>
+                <h3 class="features_h3">Guaranteed to work in par casesm.</h3>
```
Comment (Aniruddha-Shriwant) [inline index.html]: What's this per "casesm" ?

**Review by Aniruddha-Shriwant — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (OmkarPh) [reply index.html]: I forgot 🙃 Maybe some typo 😁

**Comment by stealzit** [comment]
Isko approve kar ke close kar de :-)

**Comment by stealzit** [comment]
merge mat kar chalega haha

**Review by Aniruddha-Shriwant — APPROVED** [review]

**Pull request merged**